### ISSUE TITLE
Add option to make the search also search comments

### ISSUE THREAD

**Issue by JayEn84** [BookStackApp/BookStack#5148]
### Describe the feature you'd like

When using the extended search, I'd like to have an additional checkbox which is not checked by default and offers the possibility to "also search comments".

### Describe the benefits this would bring to existing BookStack users

I recently commented on a page. And while I remember parts of my comment, I cannot not remember the page's name. If it was possible to search the comments, too, I'd find the page I am looking for.

### Can the goal of this request already be achieved via other means?

I don't know of any possibility to search comments (except directly within the database).

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

### How long have you been using BookStack?

3 months to 1 year

### Additional context

![image](https://github.com/user-attachments/assets/1cf0a5b4-f15c-4245-952c-077f71b4e598)

**Comment by vmario89** [comment]
i am looking for the same feature, so i am upvoting it. i guess it should be possible due to the MariaDB structure it has.

**Comment by prohtex** [comment]
Yes please! Searchable comments (similar to wikijs) would be AMAZING!

**Comment by gaidap** [comment]
This would be great. I am currently evaluating BookStack as an option for our knowledge base system.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/5148
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
